src/client/creature.cpp

```diff
@@ -540,9 +538,8 @@ void Creature::updateWalkAnimation(int totalPixelsWalked)
             if(!self->m_walking || self->m_walkTimer.ticksElapsed() >= self->getStepDuration(true))
                 self->m_walkAnimationPhase = 0;
             self->m_walkFinishAnimEvent = nullptr;
-        }, std::min<int>(footDelay, 200));
+        }, footDelay);
```

Comment:
Can you explain the implications of this change?

Reply:
As I understand this code suppose to reset animation frame when when stop. This change makes it that there is no change in animation speed during walk and during the last frame when we stop.